README.md

```diff
@@ -142,12 +142,12 @@ Our implementation modifies the protocols described in "[The Honey Badger of BFT
 
 We have simplified algorithm naming conventions from the original paper.
 
-|  Algorithm Name  | Original Name                    |
-| ---------------- | -------------------------------- |
-| Honey Badger     | HoneyBadgerBFT                   |
-| Subset           | Asynchronous Common Subset (ACS) |
-| Broadcast        | Reliable Broadcast (RBC)         |
-| Binary Agreement | Binary Byzantine Agreement (BBA) |
+|  Algorithm Name  | Original Name                                 |
+| ---------------- | --------------------------------------------- |
+| Honey Badger     | HoneyBadgerBFT                                |
+| Subset           | Asynchronous Common Subset (ACS)              |
+| Broadcast        | Reliable Broadcast (RBC)                      |
+| Binary Agreement | Asynchronous Binary Byzantine Agreement (ABA) |
```

Comment:
Do we need the unfortunate abbreviation? It's not a standard one, is it?

This abbreviation should be ABBA.

Reply:
I think the table is mainly to make it easier to connect the paper with the code. And "ABA" is used quite a few times throughout the paper.